boards/v2/intel/intel_ish/Kconfig.defconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if TEST
+config TEST_EXTRA_STACK_SIZE
+	int
```

Comment:
`int` should not be here as the Kconfig is created in zephyr itself

Reply:
removed